message_definitions/v1.0/common.xml

```diff
@@ -1137,11 +1137,11 @@
       </entry>
       <entry name="MAV_CMD_DO_MOTOR_TEST" value="209">
         <description>Mission command to perform motor test</description>
-        <param index="1">motor sequence number (a number from 1 to max number of motors on the vehicle)</param>
+        <param index="1">motor number (a number from 1 to max number of motors on the vehicle)</param>
         <param index="2">throttle type (0=throttle percentage, 1=PWM, 2=pilot throttle channel pass-through. See MOTOR_TEST_THROTTLE_TYPE enum)</param>
         <param index="3">throttle</param>
         <param index="4">timeout (in seconds)</param>
-        <param index="5">Empty</param>
+        <param index="5">motor test type (See MOTOR_TEST_TYPE enum)</param>
```

Comment:
This is what @lorenz opposed to when I suggested this approach (https://github.com/mavlink/qgroundcontrol/issues/3850#issuecomment-240167134).... Because if the other side is not zero-initializing the struct before sending it would request the wrong thing.  Thus I suggested the other way of interpreting the motor id:

```
+        <param index="1">motor sequence number ([1, max_motors]: test motors in a vehicle-dependent sequence that follows a clockwise order, usually shown as A, B, C, D, etc; [129, 129 + max_motors]:test motors following the normal channel order</param>
```

I actually think it's very reasonable to say that programs have to zero-initialize the struct and not doing that is an actual bug. However see the comment there.